modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/chatwoot.jsp

```diff
@@ -16,24 +16,27 @@
 
 <%@ include file="/dynamic_include/init.jsp" %>
 
-<script>
-	(function (d, t) {
-		var BASE_URL = 'https://app.chatwoot.com';
+<script type="text/javascript">
+	if (!(window.$chatwoot && window.$chatwoot.hasLoaded)) {
```

Comment:
I have a feeling that without the `?` it may cause some error, does it makes sense?

```suggestion
	if (!(window.$chatwoot && window.$chatwoot?.hasLoaded)) {
```

Also it could be (prefered)

```suggestion
	if (!window.$chatwoot?.hasLoaded) {
```

Reply:
Makes sense to simplify! Changes pushed.

Reply:
The issue is that now I have a source format problem:
![image](https://github.com/pt-liferay-solutions/liferay-portal/assets/59687465/53ab0d14-b003-4662-b7e0-a99ece7638b1)

Reply:
I changed back to the original form, as there is a source format when using ? in jsp